include/KokkosInterface.h

```diff
@@ -15,15 +15,17 @@ namespace sierra {
 namespace nalu {
 
 using HostSpace = Kokkos::HostSpace;
-using DeviceSpace = Kokkos::DefaultExecutionSpace;
+//!!!! Important: the 'Kokkos::Serial' here makes sure we only run serial. !!!!
+// using DeviceSpace = Kokkos::DefaultExecutionSpace;
+using DeviceSpace = Kokkos::Serial;
```

Comment:
Is there a situation where Kokkos::Serial isn't enabled? Can you give me an example?